### ISSUE TITLE
How to get HTML From Url

### ISSUE THREAD

**Issue by Conghdos** [pichillilorenzo/flutter_inappbrowser#2]
I want get HTML of page, but I call this.injectScriptCode("""document.body.innerHTML""")=>void??
How to what to make it!
Thank!

**Comment by pichillilorenzo** [comment]
I'm going to add the possibility of return a value from evaluation of javascript in the next release! I think it will be released tomorrow or the next day!

**Comment by pichillilorenzo** [comment]
Released new version **v0.1.1**.
Now you can use something like that inside your `InAppBrowser` class to get the html:
```dart
  @override
  Future onLoadStop(String url) async {
    // get body html
    var html = await this.injectScriptCode("document.body.innerHTML");
  }
```
